pages/common/vboxmanage-movevm.md

```diff
@@ -0,0 +1,12 @@
+# vboxmanage-movevm
+
+> Move a virtual machine (VM) to a new location on the host system.
+> More information: <https://www.virtualbox.org/manual/ch08.html#vboxmanage-movevm>.
+
+- Move the specified VM to the current location:
+
+`VBoxManage movevm {{vm_name}}`
+
+- Specify the new location (full or relative pathname) of the VM:
+
+`VBoxManage movevm {{vm_name}} --folder {{path/to/new_location}`
```

Comment:
```suggestion
`VBoxManage movevm {{vm_name}} --folder {{path/to/new_location}}`
```

Thanks :) One more change and this looks good to me

Reply:
Ah my bad, will change it. I will do it more conscientiously on the next page!